packages/dashboards/addon/templates/components/report-actions/add-to-dashboard.hbs

```diff
@@ -18,8 +18,7 @@
         class="text-input dashboard-title"
         autofocus=true
       }}
-      {{!-- template-lint-disable link-href-attributes --}}
-      <a class="dashboard-action-text" role="button" onclick={{action (toggle "shouldCreateDashboard" this)}}>Select from my dashboards</a>
+      <div class="dashboard-action-text" role="button" onclick={{action (toggle "shouldCreateDashboard" this)}}>Select from my dashboards</div>
```

Comment:
Can we make these buttons, instead of divs with role `button`?

Reply:
We apply styles to button elements so that would require overriding our default button styles for this one case. We've done this in the past where we want some text to have an action attached to it.

Reply:
I get that, but we should rethink the accessibility of this. Its just more work to make divs more accessible, we would have to add aria attributes, etc

Reply:
I agree that using buttons would be better, but that's a bit more than I wanted to accomplish in this PR. This was just meant to be a quick win so we could get rid of the template lint disable lines. I think we should open an issue to set navi-wide default button styles that would change depending on a class set on the button. Say, default button elements would keep their current styles, but if you apply a class like `text-only` or something it would remove any borders.

Reply:
But my point is that going from an anchor to a div is not exactly a win.

Reply:
It's a win that we can remove the template lint disable lines. We've got other places that use `role="button"` so we should move these and the rest of those to button elements in a separate issue.

Reply:
Seems like `role="button"` does add some accessibility features to the element
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role